mailpoet/lib/Migrator/Migrator.php

```diff
@@ -62,25 +62,27 @@ public function run(Logger $logger = null): void {
    * 1. Db migrations ordered by filename
    * 2. App migrations ordered by filename
    * 3. Unknown migrations (saved in store but not in repository e.g., renamed or deleted)
-   * @return array{name: string, status: string, started_at: string|null, completed_at: string|null, retries: int|null, error: string|null, unknown: bool}[]
+   * @return array{name: string, level: string|null, status: string, started_at: string|null, completed_at: string|null, retries: int|null, error: string|null, unknown: bool}[]
```

Comment:
Changing this probably also affects types in the [Logger](https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/lib/Migrator/Logger.php). I wonder if it can be better reused, but not sure. [PHPStan type aliases](https://phpstan.org/writing-php-code/phpdoc-types#local-type-aliases) could work for PHPStan, but I don't know if the IDEs support that too, so maybe it's safer to keep it copy-pasted.